README.md

```diff
@@ -1,5 +1,102 @@
-tbench
+yab
 ======
 
-tbench is a tool to benchmark Thrift services. It currently supports
-TChannel as a transport, but will also support HTTP.
+`yab` (Yet Another Benchmarker) is tool to benchmark Thrift services. It has
```

Comment:
Perhaps: "to benchmark YARPC services".

Then you can just add `raw` and `json` to the TODO :)